src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -106,13 +101,6 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medFilteringAbstractT
 
     QObject::connect ( d->filters, SIGNAL ( currentIndexChanged ( int ) ), this, SLOT ( onFiltersActivated ( int ) ) );
 
-    QLabel * dataTypeLabel = new QLabel ( tr ( "Data type :" ) );
```

Comment:
After thinking about that: meeeeh. I'm not sure that it would be really interesting to do a specific toolbox for that. You said, when we spoke about that few weeks ago, that you liked to be able to test your data. If it's a dev tool, it can be done in the code, you can already test the type of data and put a log. @fcollot ?